packages/neuron-wallet/src/channel/index.ts

```diff
@@ -127,7 +128,7 @@ const listenToChannel = () => {
     setTimeout(() => {
       e.sender.send(Channel.GetCellsByTypeHash, {
         status: 1,
-        result: [cell],
+        result: ['cells'],
```

Comment:
Why not return response of `getUnspentCells`

Reply:
Not used now, not important